docs/Getting-Started-TPU-VM.md

```diff
@@ -199,3 +201,20 @@ gcloud compute tpus tpu-vm ssh $NAME --zone $ZONE --worker=all --command 'sudo r
 
 **and then you have to ctrl-c this after about 10 seconds**. Otherwise, gcloud will think the command failed and will
 try again, and get stuck in a loop forever. (You can ctrl-c it at any point after 10 seconds.)
+
+### Docker-related Issue
+
+If you get a `permission denied` error with your `docker.sock`. You should be able to fix with:
+
+```shell
+sudo usermod -aG docker $USER
+sudo chmod 666 /var/run/docker.sock
```

Comment:
This second line shouldn't be needed, but you may need to restart the shell to have the `usermod` above take effect.

(I'd recommend we not suggest the `chmod 666` as it would make the host less secure.)

Reply:
this is on the launching machine right? should say so if so